src/leiningen/ezbake.clj

```diff
@@ -26,6 +26,7 @@ Actions:
                   ezbake-core/resource-path)]
         (lein-ezbake-core/prepare-resource-dir project)
         (try
+          (ezbake-core/validate! project)
```

Comment:
@haus had suggested doing the validation of the `vars` from the project config via a parameter to the `make-template-map` function.  That would seem like a reasonable way to go but I thought it might be even better to just have an explicit validation step before any of the package building process has started -- setup of the `./target` directory, etc. -- where any invalid config could cause the package build to fail.